docs/ui/interactions/fragments/web/customization.md

```diff
@@ -0,0 +1,33 @@
+## Customization
+
+### Slots
+
+Amplify UI Components provides the following [slot](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot).
+
+| Name                 | Description                         |
+| -------------------- | ------------------------------------|
+| `Header`             | Content placed at the top of the UI |
```

Comment:
For now this works. We have access to this documentation via the docs.json: https://github.com/aws-amplify/docs/blob/0682ec157a414cc6bfe096979be4d652f9fd2c9a/client/src/docs-ui/ui-component-props/ui-component-props.tsx#L2 but we can save this for our GA launch